src/sentry/data_secrecy/models/data_access_grant.py

```diff
@@ -0,0 +1,57 @@
+from enum import StrEnum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, control_silo_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@control_silo_model
+class DataAccessGrant(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class GrantType(StrEnum):
+        ZENDESK = "zendesk"
+        MANUAL = "manual"
+
+    class RevocationReason(StrEnum):
```

Comment:
Django also provides `django.db.models.TextChoices` which gives you an Enum with integration into `choices` fields.